acceptance/indy_acceptance/indy-tests/libraries/constant.py

```diff
@@ -0,0 +1,60 @@
+
+"""
+Created on Nov 9, 2017
+
+@author: khoi.ngo
+
+Containing all constants that are necessary to execute test scenario.
+"""
+
+
+class Colors:
+    """
+    Class to set the colors for text.
+    Syntax:  print(Colors.OKGREEN +"TEXT HERE" +Colors.ENDC)
+    """
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'  # Normal default color.
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+class Roles:
+    """
+    Class to define roles.
+    """
+    TRUSTEE = "TRUSTEE"
+    STEWARD = "STEWARD"
+    TRUST_ANCHOR = "TRUST_ANCHOR"
+    TGB = "TGB"  # obsolete.
+    NONE = ""
+
+
+class Constant:
+    """
+    Class Constant store some necessary paths.
+    """
+    import os
+    user_home = os.path.expanduser('~') + os.sep
+    work_dir = user_home + ".indy"
+    seed_default_trustee = "000000000000000000000000Trustee1"
+    # The path to the genesis transaction file is configurable. The default directory is "/var/lib/indy/sandbox/".
+    genesis_transaction_file_path = "/var/lib/indy/sandbox/"
```

Comment:
This path is the node path to `sandbox` pool. So it does not relate to the acceptance tests or libindy. By default we should use the pool transactions genesis file bundled with the acceptance tests. It is `pool.txn` from `indy-node/acceptance` directory that must be the content root.
However, `pool.txn` must be updated - BLS keys of nodes must be added to it. Also it makes sense to change IP addresses of nodes in `pool.txn` to ones being used in Docker pool.

Reply:
This issue has not been fixed.

Reply:
Please fix this.

Reply:
Updated. Pointed to pool.txn, indy-node/acceptance directory .

Reply:
Please update also the content of the file `pool.txn` as mentioned above.

In order to do this you can run a local pool in Docker and a client for it using https://github.com/evernym/sovrin-environments. See https://github.com/evernym/sovrin-environments/blob/stable/docker/README.md for details. Use the defaults for the pool and client for it.
Then open a new terminal window and log in to the running client container using the command `docker exec -it sovrinclient bash`.
In the client container open the file `/home/indy/.indy-cli/networks/sandbox/`. The current content of `indy-node/acceptance/pool.txn` should be replaced with the content of this opened file.